src/simplnx/Utilities/SegmentFeatures.cpp

```diff
@@ -147,7 +147,7 @@ Result<> SegmentFeatures::execute(IGridGeometry* gridGeom)
     if(std::chrono::duration_cast<std::chrono::milliseconds>(now - start).count() > 1000)
```

Comment:
THis should be swapped out for the MessageHelper and ThrottledMessenger classes.